app/javascript/components/cloud-network-form/cloud-network-form.schema.js

```diff
@@ -1,9 +1,19 @@
 import { componentTypes, validatorTypes } from '@@ddf';
 import { API } from '../../http_api';
 
-function createSchema(ems, cloudNetworkId, loadSchema, providerFields = []) {
-  const providers = ems.filter((tenant) => tenant.type !== 'ManageIQ::Providers::Nuage::NetworkManager');
+let stop = true;
```

Comment:
What is the purpose of the `stop`?

Reply:
That is from older code, its removed now just need to fix test cases and push changes